.changeset/serious-roses-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+'example-app': patch
+'@backstage/plugin-api-docs': patch
```

Comment:
This actually looks like it should be a `minor` release, as the API has changed. `bind` now requires `registerComponent` instead of `createComponent`.

Reply:
Bumped to `minor` with a bit more detail on that change.